README.md

```diff
@@ -4,7 +4,11 @@ Pulp 3 Ansible installer
 The Pulp 3 Ansible installer consists of a collection of roles. Each role installs and configures a
 component of Pulp. The roles are not currently available on Ansible Galaxy; to run the Pulp 3
 Ansible installer, the [ansible-pulp](https://github.com/pulp/ansible-pulp) git repository must
-be cloned.
+be cloned or downloaded.
+
+This version of the installer, 3.0.1, installs Pulp 3.1.0 specifically.
```

Comment:
Should we focus on 3.1.0 since that is what is about to go out?

Reply:
There were breaking changes between 3.0.1 and 3.1.0, correct?

And even if not, I think some plugins say they need 3.0.x, not 3.1.x.

So we need to do a release for 3.0.1, then a 3.1.0 release, for the sake of affected users.

Also, at the time I submitted this PR, 3.1.0 was not out yet for me to test the changes.

Reply:
Plugins are advised to lock onto a 3.y release so the current state of plugin dependencies that are released are to depend on 3.0 <= pulpcore < 3.1. So when pulpcore releases all plugins will need to release compatibility releases at the least.

Reply:
We can continue with 3.0.1 for now, and make another release/tag for 3.1.0 when it's available. I'm building it now.

Reply:
@mikedep333 shouldn't the installer version 3.0.1 install 3.0.1?

Reply:
My bad. Fixed the comment.